src/frontend/src/storage/index.test.ts

```diff
@@ -311,6 +312,70 @@ test(
     vi.useRealTimers();
   })
 );
+
+test(
+  "should retrieve last anchor",
+  withStorage(async () => {
+    const origin = "https://example.com";
+    const principal = Principal.fromText("2vxsx-fae");
+    await setKnownPrincipal({
+      userNumber: BigInt(10000),
+      origin,
+      principal,
+    });
+
+    expect(await getAnchorIfLastUsed({ principal, origin })).toBe(
+      BigInt(10000)
+    );
+  })
+);
+
+test(
+  "should not retrieve anchor if not most recent",
+  withStorage(async () => {
+    const origin = "https://example.com";
+    const principal1 = Principal.fromText(
+      "hawxh-fq2bo-p5sh7-mmgol-l3vtr-f72w2-q335t-dcbni-2n25p-xhusp-fqe"
+    );
+    vi.useFakeTimers().setSystemTime(new Date(0));
+    await setKnownPrincipal({
+      userNumber: BigInt(10000),
```

Comment:
Can we put `BitInt(10000)` and `BigInt(1000)` in variables to better read the expectations?